<launch>
  <node name="siamfc_initialiser" pkg="siamfc_test" type="siam_fc_initialiser.py" output="screen">
    <!-- <remap from="/input_image_topic" to="/camera/image_rectified"/> -->
    <remap from="/input_image_topic" to="/usb_cam/image_raw"/>
  </node>
</launch>